src/main/java/com/datapipe/jenkins/vault/credentials/common/VaultHelper.java

```diff
@@ -76,100 +110,20 @@ static String getVaultSecretKey(@NonNull String secretPath,
-            }
-
-            VaultConfiguration configuration = null;
-            for (VaultConfigResolver resolver : ExtensionList.lookup(VaultConfigResolver.class)) {
-                if (configuration != null) {
-                    configuration = configuration
-                        .mergeWithParent(resolver.getVaultConfig(credentialContext));
-                } else {
-                    configuration = resolver.getVaultConfig(credentialContext);
-                }
-            }
-
-            if (configuration == null) {
-                throw new IllegalStateException("Vault plugin has not been configured.");
-            }
-
-            configuration.fixDefaults();
-            if (engineVersion == null) {
-                engineVersion = configuration.getEngineVersion();
-            }
-
-            String msg = String.format(
-                "Retrieving vault secret path=%s engineVersion=%s",
-                secretPath, engineVersion);
-            LOGGER.info(msg);
-
-            try {
-                VaultConfig vaultConfig = configuration.getVaultConfig();
-
-                if (prefixPath != null) {
-                    vaultConfig.prefixPath(prefixPath);
-                }
-
-                if (namespace != null) {
-                    vaultConfig.nameSpace(namespace);
-                }
-
-                VaultCredential vaultCredential = configuration.getVaultCredential();
-                if (vaultCredential == null) vaultCredential = retrieveVaultCredentials(configuration.getVaultCredentialId(), credentialContext);
-
-                VaultAccessor vaultAccessor = new VaultAccessor(vaultConfig, vaultCredential);
-                vaultAccessor.setMaxRetries(configuration.getMaxRetries());
-                vaultAccessor.setRetryIntervalMilliseconds(configuration.getRetryIntervalMilliseconds());
-                vaultAccessor.init();
-
-                return vaultAccessor.read(secretPath, engineVersion).getData();
-            } catch (VaultPluginException vpe) {
-              throw vpe;
-            } catch (Exception e) {
-                throw new RuntimeException(e);
-            }
-        }
-    }
-
     private static VaultCredential retrieveVaultCredentials(String id, ItemGroup<Item> itemGroup) {
         if (StringUtils.isBlank(id)) {
             throw new VaultPluginException(
-                "The credential id was not configured - please specify the credentials to use.");
+                    "The credential id was not configured - please specify the credentials to use.");
         } else {
             LOGGER.log(Level.INFO, "Retrieving vault credential ID : " + id);
         }
         List<VaultCredential> credentials = CredentialsProvider
-            .lookupCredentials(VaultCredential.class,
-                itemGroup,
-                ACL.SYSTEM,
-                Collections.<DomainRequirement>emptyList());
+                .lookupCredentials(VaultCredential.class,
```

Comment:
(whitespace-only changes)

Reply:
Continuation indent size according to codestyle is set to `4` and this seems like a `8`
https://github.com/jenkinsci/hashicorp-vault-plugin/blob/0bffd1b067452734d4aca451eeeff2d890f2ce61/.idea/codeStyles/Project.xml#L6

Reply:
I do not care at all what the value is, just noting as an aside that the diff is a bit harder to read due to gratuitous hunks.